onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -5319,6 +5358,29 @@ def aten_mul_bool(self: BOOL, other: BOOL) -> BOOL:
     return op.And(self, other)
 
 
+@torch_op(("aten::mul", "aten::mul.Tensor", "_operator::mul"))
```

Comment:
This is missing complex=True? Making regression on CI failing https://github.com/pytorch/pytorch/pull/112758